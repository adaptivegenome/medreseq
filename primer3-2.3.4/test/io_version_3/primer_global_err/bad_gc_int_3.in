PRIMER_SEQUENCE_ID=bad_gc_int_3
SEQUENCE=TAGCTGATGCTAGCTAGCGTGGGCATGCTAGCTAGCTGATGCAGCTAGCTGATCGTAGCTAGCTATCGATCGTAGCTGACTGATCGTAGCTAGCTGATCGATGCTAGCTAGCTAGCTGATCATCTACGTAGCTGACTATCGATGCTACTACTATCGATCGTAGCTAGCTGATCGTAGCTAGCTAGCTGATGCTAC
PRIMER_INTERNAL_OLIGO_MAX_GC=60
PRIMER_INTERNAL_OLIGO_MIN_GC=70
=
