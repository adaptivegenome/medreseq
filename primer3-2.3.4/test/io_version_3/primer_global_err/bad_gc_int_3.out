PRIMER_SEQUENCE_ID=bad_gc_int_3
SEQUENCE=TAGCTGATGCTAGCTAGCGTGGGCATGCTAGCTAGCTGATGCAGCTAGCTGATCGTAGCTAGCTATCGATCGTAGCTGACTGATCGTAGCTAGCTGATCGATGCTAGCTAGCTAGCTGATCATCTACGTAGCTGACTATCGATGCTACTACTATCGATCGTAGCTAGCTGATCGTAGCTAGCTAGCTGATGCTAC
PRIMER_INTERNAL_OLIGO_MAX_GC=60
PRIMER_INTERNAL_OLIGO_MIN_GC=70
PRIMER_ERROR=Illegal value for PRIMER_INTERNAL_OLIGO_GC
=
